cmd/openshift-tests/disaster-recovery.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/kubernetes/pkg/kubectl/util/templates"
+
+	"github.com/openshift/origin/pkg/test/ginkgo"
+	_ "github.com/openshift/origin/test/e2e/dr"
+)
+
+// disasterRecoverySuites are all known disaster recovery test suites this binary should run
+var disasterRecoverySuites = []*ginkgo.TestSuite{
+	{
+		Name: "all",
+		Description: templates.LongDesc(`
+		Run all tests.
+		`),
+		Matches: func(name string) bool {
+			return strings.Contains(name, "[Feature:DisasterRecovery]")
+		},
+		TestTimeout: 120 * time.Minute,
+	},
+}
+
+// DisasterRecoveryOptions lists all options for disaster recovery tests
+type DisasterRecoveryOptions struct {
+	Suite       string
+	TestOptions []string
+}
+
+func (o *DisasterRecoveryOptions) OptionsMap() (map[string]string, error) {
+	options := make(map[string]string)
+	for _, option := range o.TestOptions {
+		parts := strings.SplitN(option, "=", 2)
+		if len(parts) != 2 {
```

Comment:
Maybe a switch here to handle part validation?

Reply:
Fixed in  bf696136c5842a2bf107534bd903ba479eb8352e